pytorch_lightning/accelerators/accelerator.py

```diff
@@ -223,23 +217,6 @@ def validation_step_end(self, output):
         """
         return self.training_type_plugin.validation_step_end(output)
 
-    def predict(self, args):
```

Comment:
@tchaton we had multiple declarations of this method inside a single class and after removing the second one TPU tests are failing... mind have look at it...

Reply:
Yes, I will